### ISSUE TITLE
Different colors for different projects

### ISSUE THREAD

**Issue by evandro-mota** [g1eb/angular2-calendar-heatmap#29]
Is it possible to have something like [this](https://d585tldpucybw.cloudfront.net/sfimages/default-source/component-pages/angular/angular-scheduler-events.jpg?sfvrsn=a0757ae4_0)?
The year overview could still be monochromatic, but when showing specific projects like in week and day overview, it would be nice if they could have different colors. Maybe doing something like below..

```
"details": [{
    "name": "Project 1",
    "date": "2016-01-01 12:30:45",
    "value": 9192,
    "color": '#ff4500'
  }]
```

**Comment by g1eb** [comment]
Absolutely! you could easily add this to the [week](https://github.com/g1eb/angular2-calendar-heatmap/blob/5744258398c9c804b750d0bee7e3048c49ac64e1/src/calendar-heatmap.component.ts#L1367) or [day](https://github.com/g1eb/angular2-calendar-heatmap/blob/5744258398c9c804b750d0bee7e3048c49ac64e1/src/calendar-heatmap.component.ts#L1550) overviews.

I'm not sure how much interest there is from others for this feature to be part of the library (and to be honest I don't want to update all versions of the heatmap like react etc). You can fork this repo and customize it for your project though.
